src/vs/workbench/services/editor/common/editorOverrideService.ts

```diff
@@ -139,22 +139,13 @@ export interface IEditorOverrideService {
 		createDiffEditorInput?: DiffEditorInputFactoryFunction
 	): IDisposable;
 
-	/**
-	 * Populates the override field of the untyped editor input
-	 * @param editor The editor input
-	 * @returns If one is populated whether or not there was a conflicting default, else undefined
-	 */
-	populateEditorId(editor: IUntypedEditorInput): Promise<{ conflictingDefault: boolean } | undefined>
-
 	/**
 	 * Given an editor determines if there's a suitable override for it, if so returns an IEditorInputWithOptions for opening
 	 * @param editor The editor to override
-	 * @param options The current options for the editor
-	 * @param group The current group
-	 * @param conflictingDefault Whether or not to show the conflicting default prompt
+	 * @param preferredGroup The group you want to open the editor in
 	 * @returns An IEditorInputWithOptionsAndGroup if there is an available override or a status of how to proceed
 	 */
-	resolveEditorInput(editor: IUntypedEditorInput, group: IEditorGroup, conflictingDefault?: boolean): Promise<ReturnedOverride>;
+	resolveEditorInput(editor: IEditorInputWithOptions | IUntypedEditorInput, preferredGroup: IEditorGroup | number | undefined): Promise<ReturnedOverride>;
```

Comment:
@lramos15 shouldn't this type be `PreferredGroup` for `preferredGroup`?

Reply:
They are the same, but there is a layering issue with preferredGroup being in browser

Reply:
I moved it out to `common`.